onnx/cpp2py_export.cc

```diff
@@ -39,6 +39,47 @@ static std::string ProtoBytesToText(const py::bytes& bytes) {
   return ProtoToString(proto);
 }
 
+template<typename T, typename Ts = typename std::remove_const<T>::type>
+std::pair<std::unique_ptr<Ts[]>, std::unordered_map<std::string, T*>> ParseProtoFromBytesMap(
+  std::unordered_map<std::string, py::bytes> bytesMap)
+{
+  std::unique_ptr<Ts[]> values(new Ts[bytesMap.size()]);
+  std::unordered_map<std::string, T*> result;
+  size_t i = 0;
+  for(auto kv : bytesMap) {
+    ParseProtoFromPyBytes(&values[i], kv.second);
+    result[kv.first] = &values[i];
+    i++;
+  }
+  return make_pair(move(values), result);
+}
+
+std::vector<py::bytes> CallInferenceFunction(
```

Comment:
nit: for various functions it might be useful to explicitly indicate it is a node-level inference (vs. graph-level inference) in the name, eg: `CallNodeInferenceFunction` and `infer_types_for_node`.

Reply:
That's a fair idea, I'll adapt to this in the next iteration.